src/core/Jalasoft.TeamUp.Resumes.Core/SkillsService.cs

```diff
@@ -18,7 +18,26 @@ public SkillsService(ISkillsRepository skillsRepository)
 
         public Skill[] GetSkills(string name)
         {
-            return this.skillsRepository.GetSkills(name).ToArray();
+            try
+            {
+                var skills = this.skillsRepository.GetSkills(name);
+                if (skills.Count() != 0)
+                {
+                    return skills.ToArray();
+                }
+                else
+                {
+                    throw new ResumeException(ErrorsTypes.NotFoundError, new Exception());
+                }
+            }
+            catch (ResumeException ex)
```

Comment:
after double-checking with Raul, the agreement is to don't have "try catch" at service level, please could you remove it